packages/geospatial/package.json

```diff
@@ -12,7 +12,8 @@
     "@mapbox/mapbox-gl-draw": "^1.4.3",
```

Comment:
Looks like this was in there prior to this PR, but is the `mapbox-gl-draw` license compatible with MIT or BSD?

Reply:
Good question. Looks like `mapbox-gl-draw` is published under the "ISC License", which is equivalent to the BSD 2-Clause and MIT licenses.